content/doc/book/managing/plugins.adoc

```diff
@@ -172,6 +172,24 @@ By default, the Jenkins controller will check for updates from the Update Center
 once every 24 hours. To manually trigger a check for updates, simply click on
 the *Check now* button in the *Updates* tab.
 
+=== List Installed Plugins and Their Versions
+
+The simplest way to retrieve the list of installed plugins and their versions is by using the Jenkins Script Console.
+
+Follow these steps:
+
+1. Open the Jenkins link:https://www.jenkins.io/doc/book/managing/script-console/[Script Console]:
```

Comment:
We use site local links so that content developers can run the site locally without being unexpectedly taken to the production web site.
```suggestion
1. Open the Jenkins link:/doc/book/managing/script-console/[Script Console]:
```